hasura/migrations/staging/1640679052192_custom_views/up.sql

```diff
@@ -0,0 +1,13 @@
+
+
+CREATE VIEW public.circle_private AS
+  SELECT id AS circle_id, discord_webhook
+    FROM circles;
+
+CREATE VIEW public.gift_private AS
+ SELECT recipient_id, sender_id, note
+   FROM token_gifts;
```

Comment:
Do we only need these columns? what about the amount of tokens?

Reply:
so in the pending_gifts query you can access the amount of tokens, for the gift_private view i've added custom permissions to check if you are the sender/recipient in order to access this view to privatise the note field